### PR TITLE
support multiple network interfaces

### PR DESCRIPTION
**Pull request by joshmarinacci**
This uses the standard Node 'os' module to get the list of network interfaces, then it invokes the existing code on each interface, collecting the results.  This helps when the user has multiple interfaces, such as when a Pi is plugged directly into the ethernet port of a computer on wifi.

### CODE REVIEW

File: pi_finder/package/finder.js [hunk 1]
```diff
@@ -42,14 +43,44 @@ proto.whitelist = [
...
+  //filter out loopback
+  subnets = subnets.filter(function(n) { return n !== '127.0.0.1'; });
+  this.total_ips = [];
+  this.doSubnet(subnets,cb);
+};
 
+//recursive function to process each subnet
+proto.doSubnet = function(subnets,cb) {
+  var addr = subnets.shift();
+  this.subnet = addr.substr(0, addr.lastIndexOf('.')) || false;
```
Comment (toddtreece) [inline pi_finder/package/finder.js]: I might tweak this a bit in the future since we no longer are just using one subnet.

**Comment by toddtreece** [comment]
Looks good, and works as expected when testing. Merging now.

Thanks for the help!

File: pi_finder/package/finder.js (hunk 1 above)
Comment (joshmarinacci) [inline pi_finder/package/finder.js]: Yeah. I expect we'd eventually rewrite this whole area with promises to make the logic clearer, but for the immediate future I wanted to make as small of a change as possible.

**Comment by joshmarinacci** [comment]
Good. I don't have a linux or windows machine to test on. Does it work there?

**Pull request merged**